packages/resolve-scripts/src/configs/webpack.client.config.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
 
     plugins: [
         new webpack.DefinePlugin({
+            'process.env': JSON.stringify(process.env),
```

Comment:
The first line sets all env variables from outside, and the second one explicitly sets NODE_ENV to production or development.